pkg/secrets/provide_conjur_secrets.go

```diff
@@ -97,3 +96,55 @@ func RetryableSecretProvider(
 		return err
 	}
 }
+
+var TickerQuit = make(chan bool)
```

Comment:
These three channels don't need to be defined/declared as global variables.

To make this testable, the "ProviderDone" channel should passed in from the caller to PeriodicSecretProvider (in main.go, we'll pass it a channel that's not used for now, but in the future we can add a signal handler that can use this channel to stop the SP).

See diffs here:
https://github.com/cyberark/secrets-provider-for-k8s/commit/d61c96c091f171be0aa04be5c75a6407a3e5f092#diff-c083eb56e1c331264db79fbb96d5fbb5ecfd8f082e9dde16e1a0e28f5f0f4230L100-R119

Reply:
made suggested changes.